### PR TITLE
[Feature] Add support for attributed strings

### PR DESCRIPTION
**Pull request by melloskitten**
## Description
I added support for attributed strings, because it was needed in some of the mockups. 
Example: 

image
image

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionHotlineViewController.swift [hunk 1]
```diff
@@ -85,6 +88,16 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController {
 			]
 		)
 	}
+
+	/// Comment me.
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionHotlineViewController.swift:92]: That was probably a TODO?

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionOverviewViewController.swift [hunk 2]
```diff
@@ -269,13 +269,15 @@ private extension ExposureSubmissionOverviewViewController {
 			),
 			.identifier(
 				CustomCellReuseIdentifiers.imageCard,
+				// TODO: Add number.
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionOverviewViewController.swift:272]: ?

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionOverviewViewController.swift [hunk 3]
```diff
@@ -284,6 +286,15 @@ private extension ExposureSubmissionOverviewViewController {
 		return data
 	}
 
+	/// Comment me.
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionOverviewViewController.swift:289]: Does this need a comment?

**Review by johannesrohwer — CHANGES_REQUESTED** [review]
Good job, thanks for adding the comments to your methods. Please just update/remove the minor things i noted.

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionHotlineViewController.swift (hunk 1 above)
Comment (melloskitten) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionHotlineViewController.swift:92]: oops overlooked. thanks for pointing it out!

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionOverviewViewController.swift (hunk 3 above)
Comment (melloskitten) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionOverviewViewController.swift:289]: same as above

**Review by johannesrohwer — APPROVED** [review]

**Review by thomasaugsten — APPROVED** [review]

**Pull request merged**